js/main.js

```diff
@@ -28,6 +35,117 @@ function addButtonHandler(event) {
+  const $h2EntryTitle = document.createElement('h2');
+  const $divNotesEntry = document.createElement('div');
+  const $pNotes = document.createElement('p');
+
+  $divRow.className = 'row';
+  $divColumn.className = 'column-half';
+  $divColumn2.className = 'column-half';
+  $imgEntryImage.className = 'image';
+  $imgEntryImage.setAttribute('src', entry.photoUrl);
+  $h2EntryTitle.textContent = entry.title;
+  $pNotes.textContent = entry.notes;
+
+  $divRow.appendChild($divColumn);
+  $divRow.appendChild($divColumn2);
+  $divColumn.appendChild($imgEntryImage);
+  $divColumn2.appendChild($liEntry);
+  $liEntry.appendChild($divTitleList);
+  $liEntry.appendChild($divNotesEntry);
+  $divTitleList.appendChild($h2EntryTitle);
+  $divNotesEntry.appendChild($pNotes);
+
+  /*  This function creates the following dom tree in which a journal entry's properties are displayed:
+              <li>
+              <div class ="row">
+              <div class = "column half">
+                <img
+                  class="image"
+                  src=""
+                  alt="" />
+                </div>
+                <div class = "column-half">
+                <div>
+                  <h2>title goes here</h2>
+                </div>
+                <div>
+                  <p>Notes go here</p>
+                </div>
+                </div>
+              </li>
+        */
+  return $divRow;
+}
+
+document.addEventListener('DOMContentLoaded', loadEntryListItems);
+
+const $divDataViewEntries = document.querySelector('#entry-list');
+
+function toggleNoEntries() {
+  if (data.entries.length > 0) {
+    $pNoEntries.setAttribute('class', 'hidden');
+  } else {
+    $pNoEntries.removeAttribute('class');
+  }
+}
+
+function loadEntryListItems(event) {
+  // This function sees if there are any entries in local storage and then renders them on the entries page.
+
+  for (let i = 0; i < data.entries.length; i++) {
+    const $renderedListItem = renderEntry(data.entries[i]);
+    $divDataViewEntries.appendChild($renderedListItem);
+  }
+
+  toggleNoEntries();
+
+  viewSwap(data.view);
 }
+
+function viewSwap(viewName) {
+  // This function is called on when switching views. It looks to see what target clicked the button, and then changes data.view to reflect the anchor clicked. I gave the OR operator, as when we click the submit button, i pass the element "$entriesButton" itself as viewName. A bit backwards, but it works.
+  if (viewName.target === $entriesButton || viewName === $entriesButton) {
+    data.view = 'entries';
+    $entryFormDataView.className = 'hidden';
+  } else if (
+    viewName.target === $newEntryButton ||
+    viewName === $newEntryButton
+  ) {
+    data.view = 'entry-form';
+  }
+  if (data.view === 'entries') {
+    $entryFormDataView.className = 'hidden';
+    $entriesDataView.className = '';
+  } else if (data.view === 'entry-form') {
+    $entryFormDataView.className = '';
+    $entriesDataView.className = 'hidden';
+  }
```

Comment:
You shouldn't need both sets of these `if`s and `else..if`s. You should only have 1 `if` statement like you're doing on lines 136-142. Your other `if` should be deleted